doc/cortx-minikube-k8s-installation.md

```diff
@@ -0,0 +1,295 @@
+```
+
+*Note: disk-name should be relpaced by a disk which is not used in solution.yaml or the os*
+
+
+**5.3 Deploy CORTX**
+
+```
+./deploy-cortx-cloud.sh
+```
+
+*Note: due to high resource requirement this might take longer to complete the deployment.*
+
+**This step completes CORTX installation**
+
+**5.4 Test that all pods are running and that CORTX is ready**
+
+```
+kubectl get pod
+```
+
+Below is the output of a successful deployment:
+
+```
+# kubectl get pod
+NAME                               READY   STATUS    RESTARTS      AGE
+consul-client-pxczf                1/1     Running   0             97m
+consul-server-0                    1/1     Running   0             97m
+cortx-control-5d9d48bf54-x2ltw     4/4     Running   0             95m
+cortx-data-car-7fd57f7bf5-jsbfz    4/4     Running   0             94m
+cortx-ha-5b7b9fbf84-65247          3/3     Running   1 (89m ago)   90m
+cortx-server-car-6744466b9-95tnv   5/5     Running   0             92m
+kafka-0                            1/1     Running   0             97m
+openldap-0                         1/1     Running   0             97m
+zookeeper-0                        1/1     Running   0             97m
+```
+
+Once you get the above output we need to check the cluster status as follows:
+
+```
+DataPod=`kubectl get pod | grep cortx-data- | grep Running | awk '{print $1}' | head -1`
+
+kubectl exec -i $DataPod -c cortx-hax -- hctl status
+```
+
+For e.g, On an successful deployment output from the above command should be as follows (`hax`, `s3server`, `ioservice` and `confd` are started):
+
+```
+# kubectl exec -i $DataPod -c cortx-hax -- hctl status
+Byte_count:
+    critical_byte_count : 0
+    damaged_byte_count : 0
+    degraded_byte_count : 0
+    healthy_byte_count : 0
+Data pool:
+    # fid name
+    0x6f00000000000001:0x33 'storage-set-1__sns'
+Profile:
+    # fid name: pool(s)
+    0x7000000000000001:0x50 'Profile_the_pool': 'storage-set-1__sns' 'storage-set-1__dix' None
+Services:
+    cortx-server-headless-svc-car 
+    [started]  hax        0x7200000000000001:0x29  inet:tcp:cortx-server-headless-svc-car@22001
+    [started]  s3server   0x7200000000000001:0x2c  inet:tcp:cortx-server-headless-svc-car@22501
+    cortx-data-headless-svc-car  (RC)
+    [started]  hax        0x7200000000000001:0x7   inet:tcp:cortx-data-headless-svc-car@22001
+    [started]  ioservice  0x7200000000000001:0xa   inet:tcp:cortx-data-headless-svc-car@21001
+    [started]  ioservice  0x7200000000000001:0x17  inet:tcp:cortx-data-headless-svc-car@21002
+    [started]  confd      0x7200000000000001:0x24  inet:tcp:cortx-data-headless-svc-car@21003
+
+```
+
+*Note: It may take several minutes for s3server instances to move from "offline" to "started"*
+
+*If the pods are not coming up correctly or any of the service are not getting `[started]` - check your `solution.yaml` for typos or mistakes which could result in a deployment failure.*
+
+**5.5 Destroy CORTX Cluster**
```

Comment:
```suggestion
**5.5 Delete CORTX Cluster**
```

The Alex bot doesn't like this word, and to appease it we'll be changing the name of this script. You can get ahead of us.

Reply:
Done!